contributing/DEVELOPING.md

```diff
@@ -84,15 +85,14 @@ For any public AMP document like: http://output.jsbin.com/pegizoq/quiet
 
 You can access is with the local JS at
 
-- normal sources: http://localhost:8000/max/output.jsbin.com/pegizoq/quiet
-- minified: http://localhost:8000/min/output.jsbin.com/pegizoq/quiet
+http://localhost:8000/proxy/output.jsbin.com/pegizoq/quiet
```

Comment:
Correct wording of this section to:

For any public AMP document like http://output.jsbin.com/pegizoq/quiet, you can access it with the local JS at http://localhost:8000/proxy/output.jsbin.com/pegizoq/quiet.